plugins/modules/aci_aaa_user.py

```diff
@@ -36,7 +36,7 @@
     description:
     - The name of the locally-authenticated user user to add.
     type: str
-    aliases: [ name, user ]
+    aliases: [ name ]
```

Comment:
I think `user` is valid here. Maybe add user to the spec instead?

Reply:
It conflicts with global username option that define user as an alias

Reply:
Oh I see, I forgot about the base spec.